.env.example

```
@@ -49,6 +49,15 @@ GITHUB_UNFURL_TOKEN=
 
 YOUTUBE_API_KEY=
 
+# Apple Push Notification service (worker). Optional — when unset, apns.send jobs no-op after logging.
+# Use Key ID + Team ID + .p8 contents (PEM or single-line with \n). Environment must match
+# device token registration from the iOS app (sandbox for dev, production for TestFlight/App Store).
+APNS_KEY_ID=
+APNS_TEAM_ID=
+APNS_BUNDLE_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APNS_BUNDLE_ID key should go before the APNS_KEY_ID key

(UnorderedKey)